bootstrap/kubeadm/api/v1alpha4/kubeadm_types_test.go

```diff
@@ -17,14 +17,16 @@ limitations under the License.
 package v1alpha4
 
 import (
-	"encoding/json"
 	"reflect"
 	"testing"
 
+	jsoniter "github.com/json-iterator/go"
 	. "github.com/onsi/gomega"
 	"github.com/pkg/errors"
 )
 
+var json = jsoniter.ConfigCompatibleWithStandardLibrary
```

Comment:
Can we have this exported once in our own package? Do we need to be stdlib compatible here, is sorting required?